index.html

```diff
@@ -1,31 +1,50 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
+  <link rel="stylesheet" href="css/main.css">
   <title>Instagram</title>
 </head>
-<body>
-  Instagram
+<body class=".body">
```

Comment:
Classes shouldn't start with a dot in the HTML file. Only when you refer to them in the CSS